ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/private/outpost/provision/cucushift-installer-rehearse-aws-ipi-private-outpost-provision-chain.yaml

```diff
@@ -0,0 +1,47 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-private-outpost-provision
+  steps:
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: aws-provision-vpc-shared
+  - chain: aws-provision-bastionhost
+  - ref: ipi-conf-aws-custom-vpc
+  - ref: ipi-conf-aws
+  - ref: ipi-conf-aws-rootvolume
+  - ref: network-conf-aws-cluster-mtu
+  - ref: proxy-config-generate
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-install-hosted-loki
+  - ref: ipi-install-install-aws
+  - ref: ipi-install-times-collection
+  - ref: storage-conf-storageclass-set-default-storageclass
+  - ref: ipi-install-post-monitoringpvc
```

Comment:
@duanwei33 @Phaow PTAL
Removed `ipi-install-monitoringpvc`
Added  `storage-conf-storageclass-set-default-storageclass` and `ipi-install-post-monitoringpvc` after install

Reply:
looks good to me.